### PR TITLE
Fix crash in ld_mod.

### PR DESCRIPTION
**Pull request by skl131313**

### CODE REVIEW

File: src/vcbuild/ldfpu.asm [hunk 1]
```diff
@@ -182,7 +182,7 @@ FM1:    ; We don't use fprem1 because for some inexplicable
         sahf                            ; transfer to flags
         jp      FM1                     ; continue till ST < ST1
         fstp    ST(1)                   ; leave remainder on stack
-        fstp    tbyte ptr [ecx]
+        fstp    tbyte ptr [rcx]
```
Comment (UplinkCoder) [inline src/vcbuild/ldfpu.asm]: is this only compiled on 64bit ?

Comment (sprinkle131313) [inline src/vcbuild/ldfpu.asm]: Yah, MSVC doesn't allow inline 64-bit assembly. The 32-bit version is inlined.

https://github.com/dlang/dmd/blob/v2.073.0/src/root/longdouble.c#L478

Comment (UplinkCoder) [inline src/vcbuild/ldfpu.asm]: rcx only exists on 64bit really.

**Comment by rainers** [comment]
LGTM.

**Pull request merged**